src/css/style.css

```diff
@@ -688,3 +688,118 @@ textarea {
     background-position: top right;
   }
 }
+
+/* Modal styles */
+.modal-container {
+  display: none;
+}
+
+.modal {
+  position: fixed;
+  background-color: rgba(128, 128, 128, 0.9);
+  inset: 0;
+  z-index: 10000;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.modal-box {
+  max-width: min(700px, 90%);
+  background-color: var(--color-white);
+  border-radius: 8px;
+  padding: var(--size-300);
+  overflow-y: auto;
+}
+
+.modal-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+}
+
+.modal-title {
+  font-family: Poppins, sans-serif;
+  font-size: 30px;
+  font-weight: 700;
+  text-align: left;
+}
+
+.timeline {
+  display: flex;
+  gap: 10px;
+  font-size: 12px;
+  color: var(--color-nav-p);
+  margin-bottom: 20px;
+}
+
+.modal .timeline li {
+  display: flex;
+  align-items: center;
+}
+
+.modal .img-container {
+  max-width: min(600px, 95%);
+  margin-inline: auto;
+}
+
+.img-container img {
+  width: 80%;
+  display: block;
+  margin: auto;
+}
+
+.desc-box {
+  display: flex;
+  flex-direction: column;
+  padding-block: var(--size-300);
+  font-size: 16px;
+  color: var(--color-nav-p);
+}
+
+.modal .tags {
+  padding-bottom: 15px;
+  border-bottom: 0.1px solid rgba(128, 128, 128, 0.2);
+}
+
+@media screen and (min-width: 40em) {
+  .modal-box {
```

Comment:
- [ ] Kindly match the given design for the desktop view of the modal pop-up as well as this is a requirement. Kindly fix the following

1. The color of the heading and close button
2. The size of the image and the margin around it on the left and right side

Your design
![image](https://user-images.githubusercontent.com/81932478/188739361-bbce47b7-d593-4cd8-bb79-a3bd613915c7.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/188739419-866e2885-dc93-4ecb-b8bb-f58f682793f9.png)

3. The number of languages in the provided design, so instead of three make them six and they should be aligned as in the provided template.

Your design
![image](https://user-images.githubusercontent.com/81932478/188740418-c05566dc-9f54-4233-86ce-8d94142e2ad5.png)

Provided design
![image](https://user-images.githubusercontent.com/81932478/188740601-a7a64860-2414-4dcc-ab92-24f8204f225c.png)

- [ ] Optional. You can increase the length of the paragraph text in the modal so as to beautify it and make it look like the original design

Provided design
![image](https://user-images.githubusercontent.com/81932478/188741204-e6e1c9eb-3b7f-4453-8c2c-10d041a4d9f1.png)

Your design
![image](https://user-images.githubusercontent.com/81932478/188741375-7adcc6fb-2313-4681-85b4-cd9eb6c76d3e.png)

Reply:
Dear @Munsa1, as you know for this task I have to display the data for every card dynamically based on the data provided in every card and the image is also different in terms of size from the cards.
so how I can change this paragraph to be larger and change the image size?